dataline-integrations/singer/bigquery/destination/requirements.txt

```diff
@@ -0,0 +1 @@
+target-bigquery == 1.4.0
```

Comment:
https://github.com/transferwise/pipelinewise-target-bigquery#schema-changes PPW seems to support schema changes at least partially, which is nice